presto-benchto-benchmarks/pom.xml

```diff
@@ -82,6 +88,13 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
```

Comment:
Please add a comment explaining why we need to do this

Reply:
Since I have excluded the spring-core dependency above in teradata benchto-driver I have added a upgraded version of it which is compatible with java 8 so that code flow doesn't break without it

Reply:
Ditto--see above.